source/Calamari/Kubernetes/Conventions/Helm/HelmManifestAndStatusReporter.cs

```diff
@@ -40,15 +41,20 @@ public async Task StartBackgroundMonitoringAndReporting(
         {
             await Task.Run(async () =>
                            {
-                               var manifest = await PollForManifest(deployment, helmCli, releaseName, revisionNumber);
+                               if (deployment.Variables.GetFlag(SpecialVariables.ResourceStatusCheck)
```

Comment:
nit: can we get this into a variable and then use it, rather than parsing/searching variables twice?

Reply:
yea I should have done that 😅